src/server/tiered_storage.h

```diff
@@ -92,12 +99,17 @@ class TieredStorage {
   struct PerDb;
   std::vector<PerDb*> db_arr_;
 
-  absl::flat_hash_map<uint32_t, uint8_t> page_refcnt_;
+  absl::flat_hash_map<uint32_t, std::pair<unsigned, unsigned> > page_refcnt_;
   util::fb2::EventCount throttle_ec_;
   TieredStats stats_;
   size_t max_file_size_;
   size_t allocated_size_ = 0;
   bool shutdown_ = false;
+
+  float defrag_bin_util_threshold_ = 0.2;
```

Comment:
change to const

Reply:
this parameter later should be configurable in run time..

Reply:
if in a later PR you will introduce a flag instead of this var you will remove it, but for now this is a const